.github/CODEOWNERS

```diff
@@ -13,7 +13,6 @@ Cargo.lock
 Cargo.toml
 /lib/rust/ @MichaelMauderer @mwu-tow @farmaazon @kazcw @vitvakatu @Frizi
 /lib/rust/ensogl/ @MichaelMauderer @farmaazon @kazcw @vitvakatu @Frizi
-/lib/rust/parser/ @kazcw @jaroslavtulach
```

Comment:
That's removing me from the list of reviewers for changes in the parser. Shall I continue the review?